papers/whitepaper.md

```diff
@@ -0,0 +1,93 @@
+
+Our Sampling Consensus algorithm is well adapted for each of the above use cases, being highly flexible and optimized for a wide range of different scenarios. That said, we are also actively exploring different verifiability techniques, such as Trusted Execution Environments (TEEs), that will allow for even higher levels of verifiability, while also reducing the cost of verifiable execution close to native cost.
+
+That said, Web2 style applications will most likely drive the demand for the Atoma Network in the short to medium term. This derives from the fact that Web3 applications are still in their infancy in terms of AI adoption. In numbers, the total market for AI (comprising not only generative AI) is estimated be worth in the tens of trillions of dollars, worldwide. At today's time of writing, the Web3 market is estimated to be worth $2.7 trillion today. We believe that in order to drive mass adoption of Web3, it is crucial to reach parity with the type of UX and UI that Web2 applications offer, today. Most of such ease of UI and UX comes due AI integration into Web2 applications backends. Therefore, we are lead to believe that Web3 protocols will need to integrate these AI capabilities in order to onboard a larger number of users.
+
+In order to empower the next generation of applications in the intersection of Web3 and AI, both data and model privacy is of the utmost importance. Through the use of TEEs, we will not only be able to provide verifiable AI compute, but also provide full data privacy to the end user.
+
+This means that Atoma nodes will not be able to access any sensitive data, such as personal information, while also not being able to access the output generated by the AI model. Only the end user will be able to decrypt the generated output, and can have high assurance that the input to the model was not shared with any third party.
+
+Through this, users can have full control of their data, allowing for AI models to operate on sensitive data. Privacy will unlock the full potential of AI, allowing for the creation of new data centric economic revenue models as well as applications that can operate on the user's behalf without revealing anything about the user.
+
+The Atoma Network can be used by Web2 companies and builders, specially software companies that are built from the ground up to be AI-centric, fast growing AI providers (such as Anthropic, etc) and software companies committed to integrating AI into their products (think of Canvas, etc). The reason for these companies to want to operate on the Atoma Network comes from the fact that a network like Atoma is positioned to drive down the costs of GPU utilization. 
```

Comment:
The Atoma Network can be used by Web2 companies and builders, especially software companies that are built from the ground up to be AI-centric, i.e fast growing AI providers (such as Anthropic, etc) and software companies committed to integrating AI into their products (think of Canvas, etc). The reason for these companies to operate on the Atoma Network comes from the fact that Atoma is positioned to drive down the costs of GPU utilization.